components/timeline/src/Timeline.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC, ReactNode, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+
+import '../assets/Timeline.scss';
+
+type TimelineItem = {
+  /* date is used to set the datetime attribute of the <time> element so must
+  be in the ISO format for a date or time. */
+  action: string
+  by?: string
+  date: Date
+  description: ReactNode
+}
+
+export type TimelineProps = StandardProps & {
+  /* Items is a compulsory property of the Timeline component.
+  Items consists of an array of item objects, each consisting of an
```

Comment:
Usually multi-line comments should follow the old JavaDoc style:
```js
 /**
  * # items
  * 
  * More info about the `items` field. Example:
  * ```
  * items: [new TimelineItem(...)] // code-doc inception
  * ```
  * 
  *     NOTE: this parameter _must_ be provided, but can be an empty array.
  */
```
Most IDEs will expect this convention, and will honour the markdown syntax so you can style the doc nicely... it's just people that use spacemacs and other command line editors (vim) like Daniel that might not get the full benefit.

Reply:
The comments are used in the 'Props' drop down on the docs pages. I wouldn't expect markdown to work and they should probably be kept brief anyway.

There's also no need to:
- mention the prop by name (that will be obvious)
- point out that it is comulsory
- point out that it's an array of objects specifically

It would probably just be something like:
```
/** The events to be included in the timeline. */

Reply:
Note that is needs to be JSDoc: `/**`

Reply:
Changed to JSDoc in latest commit